### ISSUE TITLE
testdata/case: Git submodule update failed

### ISSUE THREAD

**Issue by wingyplus** [sourcegraph/srclib-sample#6]
Git submodule cannot update because url	of submodule was wrong.

```
Cloning into 'testdata/case/tiny-test-repo-with-readme'...
fatal: Unable to find remote helper for 'ttps'
Clone of 'ttps://github.com/sgtest/tiny-test-repo-wth-readme.git' into submodule path 'testdata/case/tiny-test-repo-with-readme' failed
```

**Comment by alexsaveliev** [comment]
There is pending PR #5. It will fix Travis issues once merged

**Comment by wingyplus** [comment]
@alexsaveliev Thanks, I will close my PR.

**Comment by alexsaveliev** [comment]
Fixed by #5
